### PR TITLE
done with rules page

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: rules.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    flex-direction: column;
+}
+
+
+
+/* footer */
+
+.footer-items {
+    display: flex;
+    /* justify-content: center; */
```
Comment (pradipgharge) [inline rules.css:151]: Don't push commented code

File: rules.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        </div>
+
+        <div class="right-side-panel">
+
+            <!-- <div class="shopping-cart" title="shopping cart"> <span class="material-icons mtrlshop navmtrl">
+                    shopping_cart
+                </span></div>
+            <div class="favorites-item" title="favorite items"><span class="material-icons mtrlshop navmtrl">
+                    favorite_border
+                </span></div> -->
```
Comment (pradipgharge) [inline rules.html:40]: commented code again

**Review by pradipgharge — COMMENTED** [review]
Code looks code, just remove the commented code!

File: rules.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    color: white;
+    background-color: purple;
+    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.75);
+    margin: 1rem;
+    cursor: pointer;
+    font-weight: bold;
+}
+
+
+
```
Comment (vanshitamoolchandani) [inline rules.css:90]: Remove extra space from here, if it's for distinction use comments

File: rules.html [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            <details>
+                <summary class="summary-head"> Fraud/Cheating </summary>
+                <p class="accordian-text">If found guilty , then your account will be blocked.</p>
+            </details>
+
+        </div>
+
+    </div>
+    <div class="brand-choice">
+        <div class="first-brand brand-logo">
```
Comment (vanshitamoolchandani) [inline rules.html:91]: Some classes are not defined in css file, you can remove it. Other than that you are good to merge!

**Review by Sahil25061999 — COMMENTED** [review]
If the colors are being repeated than use variables for them, here the silver color is being repeated.

**Pull request merged**